torch/csrc/jit/passes/xnnpack_rewrite.cpp

```diff
@@ -1,17 +1,32 @@
+
+#include <ATen/core/jit_type.h>
+#include <ATen/native/xnnpack/OpContext.h>
+
 #include <torch/csrc/jit/passes/constant_pooling.h>
 #include <torch/csrc/jit/passes/constant_propagation.h>
 #include <torch/csrc/jit/passes/xnnpack_rewrite.h>
 #include <torch/csrc/jit/passes/fuse_linear.h>
 #include <torch/csrc/jit/passes/graph_rewrite_helper.h>
+#include <torch/csrc/jit/passes/prepack_folding.h>
 #include <torch/csrc/jit/passes/subgraph_rewrite.h>
 #include <torch/csrc/jit/ir/subgraph_matcher.h>
+#include <torch/csrc/jit/ir/ir.h>
 
 namespace torch {
 namespace jit {
 
 #ifdef USE_XNNPACK
 
 namespace {
+std::unordered_set<std::string>* getFoldablePackingOps() {
```

Comment:
returning an address is pretty wired, can you just return the set?

Reply:
or define this inside the filter function

Reply:
Actually I will just revert to old style of checking. I had this thinking more ops will be added for quantization, but that is not needed with filter function.